precompiles/regular/IRegular.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.7;
```

Comment:
NIT: we now use `pragma solidity 0.8.26;` in protocol contract. Maybe we can do the same here?